### DISCUSSION TITLE
How to resize an image ?

### DISCUSSION THREAD

**Discussion by TGayant** [ShapeCrawler/ShapeCrawler#164] [Q&A] [answered]
Hi,
I add an image like on your example "picture.Image.SetImage (pic)" but I would like to be able to resize it is it possible?

**Comment by AdamShakhabov** [comment, marked as answer]
No, this is not possible at the present.

Created [https://github.com/ShapeCrawler/ShapeCrawler/issues/166](https://github.com/ShapeCrawler/ShapeCrawler/issues/166) issue for this.

_We are always happy to receive pull requests :blush:_

**Comment by AdamShakhabov** [comment]
@TGayant, [**v0.24.0**](https://github.com/ShapeCrawler/ShapeCrawler/releases/tag/v0.24.0) version has been released. Now you can resize the picture via its `Width` and `Height` properties:

```C#
using IPresentation presentation = SCPresentation.Open("test.pptx", true);
IPicture picture = presentation.Slides[0].Shapes.OfType<IPicture>().First();

picture.Width = 300; // pixels
picture.Height = 400;

presentation.Save();
```

Please, let me know if it does not work correct.
